[2022-06-24 10:24:03.827] [remoteagent] [info] Extension host agent started.
[2022-06-24 10:24:04.432] [remoteagent] [error] [127.0.0.1][ee7df092][ManagementConnection] Unknown reconnection token (never seen).
[2022-06-24 10:24:04.521] [remoteagent] [error] [/home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30]: Failed to parse /home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30/package.nls.json: Value expected.
[2022-06-24 10:24:04.523] [remoteagent] [error] [/home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30]: Failed to parse /home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30/package.nls.json: Value expected.
[2022-06-24 10:24:04.593] [remoteagent] [error] [127.0.0.1][ec32da6a][ExtensionHostConnection] Unknown reconnection token (never seen).
[2022-06-24 10:24:04.694] [remoteagent] [error] [/home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30]: Failed to parse /home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30/package.nls.json: Value expected.
[2022-06-24 10:24:28.760] [remoteagent] [info] [127.0.0.1][f7b39765][ManagementConnection] New connection established.
[2022-06-24 10:24:29.377] [remoteagent] [error] [/home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30]: Failed to parse /home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30/package.nls.json: Value expected.
[2022-06-24 10:24:29.390] [remoteagent] [error] [/home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30]: Failed to parse /home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30/package.nls.json: Value expected.
[2022-06-24 10:24:29.446] [remoteagent] [error] [/home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30]: Failed to parse /home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30/package.nls.json: Value expected.
[2022-06-24 10:24:29.721] [remoteagent] [info] [127.0.0.1][a3262087][ExtensionHostConnection] New connection established.
[2022-06-24 10:24:29.760] [remoteagent] [info] [127.0.0.1][a3262087][ExtensionHostConnection] <1386> Launched Extension Host Process.
[2022-06-24 10:24:31.036] [remoteagent] [error] [/home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30]: Failed to parse /home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30/package.nls.json: Value expected.
[2022-06-24 10:24:31.056] [remoteagent] [error] [/home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30]: Failed to parse /home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30/package.nls.json: Value expected.
[2022-06-24 10:24:32.017] [remoteagent] [error] [/home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30]: Failed to parse /home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30/package.nls.json: Value expected.
[2022-06-24 10:24:32.350] [remoteagent] [error] [/home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30]: Failed to parse /home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30/package.nls.json: Value expected.
[2022-06-24 10:24:32.921] [remoteagent] [error] [/home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30]: Failed to parse /home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30/package.nls.json: Value expected.
[2022-06-24 10:29:31.541] [remoteagent] [error] [/home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30]: Failed to parse /home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30/package.nls.json: Value expected.
[2022-06-24 10:34:31.738] [remoteagent] [error] [/home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30]: Failed to parse /home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30/package.nls.json: Value expected.
[2022-06-24 10:39:31.936] [remoteagent] [error] [/home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30]: Failed to parse /home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30/package.nls.json: Value expected.
[2022-06-24 10:44:32.192] [remoteagent] [error] [/home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30]: Failed to parse /home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30/package.nls.json: Value expected.
[2022-06-24 10:49:32.432] [remoteagent] [error] [/home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30]: Failed to parse /home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30/package.nls.json: Value expected.
[2022-06-24 10:54:32.715] [remoteagent] [error] [/home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30]: Failed to parse /home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30/package.nls.json: Value expected.
[2022-06-24 10:59:32.957] [remoteagent] [error] [/home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30]: Failed to parse /home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30/package.nls.json: Value expected.
[2022-06-24 11:04:33.460] [remoteagent] [error] [/home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30]: Failed to parse /home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30/package.nls.json: Value expected.
[2022-06-24 11:09:34.455] [remoteagent] [error] [/home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30]: Failed to parse /home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30/package.nls.json: Value expected.
[2022-06-24 11:14:34.619] [remoteagent] [error] [/home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30]: Failed to parse /home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30/package.nls.json: Value expected.
